### DISCUSSION TITLE
Index polygons

### DISCUSSION THREAD

**Discussion by mattlucht** [digital-land/data-standards-backlog#49] [Planning consideration]
Land Registry Index Polygons have been identified as a planning consideration that will add value to the PlanX service. The purpose of this discussion is to ask any questions that come up during the data modeling of HMLR index polygons and to record any design decisions that are made.

**Comment by colmjude** [comment]
## What we've looking for

We want to know more about:

* what do you use the Land Registry index polygons for?
* do you refer to them by any other names?
* do you use them to help make any decisions? What are those decisions?
* what do you think is missing from the Land Registry Index polygon data?
* Anything else we should know about them?

**Comment by colmjude** [comment]
## Draft schema

Based on the already available data, we've drafted [a schema](https://github.com/digital-land/specification/blob/main/content/dataset/title-boundary.md).

In this draft, we've called the dataset `title-boundary` because we understand that each Land Registry Index Polygons is a title's boundary or extent. We are interested to know if you think this is a sensible or confusing thing to do?

The published data includes a `VALIDFROM` field. We are keen to find out if this is the date the boundary or polygon was created or the date it was added to the index polygon dataset that gets published monthly.

**Comment by mattlucht** [comment]
To add some context about why we're looking at this. We've heard from the ODP team that users of PlanX often times struggle to create the site boundary when making their application – for householder applications it is believed that around ~98% of the required boundaries are the same as the HMLR index polygons so using this as the starting point will significantly help the performance and usability of the service.

**Comment by colmjude** [comment]
We've added [index polygons from several local authority areas](https://www.planning.data.gov.uk/dataset/title-boundary) to the platform. We ended up calling the dataset `title-boundary` because that seems to be a more descriptive name for what the records are.

We've only added data for several local authority areas to start with because that is already over 2,400,000 entities so by far the largest dataset to date. We wanted to test the platform with these first and then we'll aim to add more over time.

Please let us know if you have any thoughts and we are always interested to know how you use the data!

**Reply by colmjude** [reply]
We had a question about attribution from a user of this data.
This dataset has a particular attribution clause (attribution outlined on [conditions of use section](https://use-land-property-data.service.gov.uk/datasets/inspire#conditions)) that states you should include 
```
The polygons (including the associated geometry, namely x, y co-ordinates) are subject to Crown copyright and database rights [year of supply or date of publication] Ordnance Survey 100026316.
```
where you are using the polygons.

At this point, citing you got the data from [planning.data.gov.uk](https://www.planning.data.gov.uk/) is insufficient, we recommend you include the attribution above.
